DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -227,6 +227,7 @@ if [[ -z $RUN_IR ]] || [[ -z $RUN_DURATION ]] || [[ -z $RUN_BFIELD ]]; then
   export RUN_DURATION=`cat Duration.txt`
   export RUN_BFIELD=`cat BField.txt`
   export RUN_DETECTOR_LIST=`cat DetList.txt`
+  export WORKFLOW_DETECTORS=$(echo "$RUN_DETECTOR_LIST" | tr " " ",")
```

Comment:
@shahor02 With this I can make it work. Does that make sense?
Now, only the used detectors of this run are used and then `serene.sh` puts things together correctly.

Reply:
@benedikt-voelkel surely it will work but it redefines the `WORKFLOW_DETECTORS` set from the JDL. It is up to DPG (@chiarazampolli @noferini ) to decide if they want to have a variable detectors list going into the reconstruction in the same period. This may alter the output files from reco which in turn may affect some postprocessing / merging etc.
The alternative could be to deactivate in the MC detectors present in the WORKFLOW_DETECTORS and absent in the  RUN_DETECTOR_LIST